frontend/src/stores/user.js

```diff
@@ -9,10 +9,12 @@ const toStringFromObject = (obj) =>
     Object.keys(obj)
```

Comment:
`Object.values()` does the same as these two lines.

Reply:
this fake credentials are a goner now because we have an official way to do auth. I will remove this